packages/admin/admin/src/common/buttons/feedback/FeedbackButton.tsx

```diff
@@ -13,7 +12,7 @@ export type FeedbackButtonClassKey = "idle" | "loading" | "success" | "error" |
 
 type OwnerState = { displayState: FeedbackButtonDisplayState };
 
-const Root = createComponentSlot(LoadingButton)<FeedbackButtonClassKey, OwnerState>({
+const Root = createComponentSlot(Button)<FeedbackButtonClassKey, OwnerState>({
```

Comment:
Does this change the behavior of the component? 
Does the `loading` behavior need to be explicitly enabled now when using `Button` instead of `LoadingButton`?

Reply:
I don't think so. In my tests it looked the same: https://deploy-preview-3034.storybook.comet-dxp.com/?path=/docs/comet-admin-feedbackbutton--docs.

Reply:
The color of the tooltip is missing but i suppose that's a different issue 😄

Reply:
It's working in main, I'll have another look into this.

Reply:
Fixed: 99f6e745359f82b863af0c930667b10d7abbf8a2